bp/tests/Resource Files/testConfig-busted.json

```diff
@@ -1,9 +1,10 @@
 {
-    "app": ".",
+    "app": "/Users/khu/Library/Developer/Xcode/DerivedData/voyager-frhgmrtfxqiflycndwcgfpqkbhdy/Build/Products/Debug-iphonesimulator/LinkedIn.app",
```

Comment:
what's up with this diff? Seems extraneous to the PR.

Reply:
Yeah, I am making sure the test doesn't fail for other reasons before it gets to `runtime` config.

Reply:
Reverting the changes as they are optional.

Reply:
If they are fixing the test, please don't revert them... I just didn't know what was going on here. Maybe add a comment in the commit saying why these diffs are here.

Reply:
Of course, I wouldn't revert them if it breaks the test. I found a different fix which is to `validateConfig` like before.